manifests/server.pp

```diff
@@ -194,12 +194,21 @@
       notify  => Exec["systemd_service_${redis_name}_preset"],
     }
   } else {
+    $sysconfig_file = "/etc/sysconfig/redis_${redis_name}"
```

Comment:
This will not work on Debian/Ubuntu, because they do not have /etc/sysconfig.
Ubuntu uses /etc/default/... and the init scripts have to be modified.